.github/workflows/test-all.yaml

```diff
@@ -103,6 +103,7 @@ jobs:
         with:
           crate: cargo-tarpaulin
       - run: ./.github/set_version.sh
+        shell: bash
```

Comment:
Oh ... yeah I think we have to add id in `test-rust.yaml` as well.

Reply:
Ok, to the extent you'd be up for owning this all the way through, that would be great